pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Field.java

```diff
@@ -244,8 +248,9 @@ enum FieldType {
 		 * @param javaDefault The field type default value in Java code
 		 * @param wireType The protobuf wire type for field type
 		 */
-		FieldType(String javaType, final String javaDefault, int wireType) {
+		FieldType(String javaType, final String boxedType, final String javaDefault, int wireType) {
```

Comment:
Good catch. I've added the missing javadoc.

The reason for the `boxedType` is the fact that Java generics don't support primitive types. So for example, if one defines `map<int32, string>` in protobuf, we cannot translate it to `Map<int, String>` in Java, we have to write `Map<Integer, String>` instead because a map key is an object and not a primitive value. The current Field/FieldType implementation in PBJ doesn't provide a way to get the boxed type, so I had to add one.